core/modules/wmgr/python/dbMgr.py

```diff
@@ -205,8 +204,8 @@ def createDb():
     hostnames = ['localhost']
     for host in hostnames:
         try:
-            dbConn.execCommand0(cmd.format(dbName, user, host))
-        except db.DbException as exc:
+            dbConn.execute(cmd.format(dbName, user, host))
+        except OperationalError as exc:
```

Comment:
I am not sure how you saw that code, I DID clean it and pushed before you commented on it.